src/create_logger_stub.ts

```diff
@@ -0,0 +1,32 @@
+import {$PropertyType} from 'utility-types';
+
+// `goodeggs-server/logger`-compatible interface
+export type Logger = {
+  child(opts: {[key: string]: unknown}): Logger;
+  debug(...params: Array<unknown>): void;
+  error(...params: Array<unknown>): void;
+  fatal(...params: Array<unknown>): void;
+  info(...params: Array<unknown>): void;
+  trace(...params: Array<unknown>): void;
+  warn(...params: Array<unknown>): void;
+};
+
+/**
+ * Creates a goodeggs-server/Logger-compatible stub.
+ */
+const stubLogger = (overrides?: Partial<Logger>): Logger => {
```

Comment:
Since there's an unrelated method in this repo called `stubLogger` and this file is `create_logger_stub`, could we rename this `createLoggerStub`?